### ISSUE TITLE
Module quit working

### ISSUE THREAD

**Issue by SdeGeata** [jclarke0000/MMM-MyWastePickup#13]
Hello;

   So the module quit working, don't know for how long. I haven't done any updates or anything, so I don't know what prompted it quitting. It just hung on 'Loading' and nothing happened.

   I removed and reinstalled and now it hangs on 'No Schedule', despite there being a custom-schedule there.

Here's the config:

`{
  	module: 'MMM-MyWastePickup',
  	position: 'bottom_left',
  	header: 'Waste Collection',
	classes: "default everyone", // for Facial Recognition
        config: {
  	collectionCalendar: 'Custom',
	weeksToDisplay:  '1',
  }
},`

**Comment by nitewing76** [comment]
It worked for me.  Did you verify the filename is: schedule_custom.csv? (I made that mistake at first) If the filename is correct...make a backup of your config, then trying this module by itself.

**Comment by SdeGeata** [comment]
Yeah, its not the file name. As I said, it quit after the update. My schedule_custom.csv was untouched. Very strange.

**Comment by nitewing76** [comment]
When I was doing game mods, if something broke we took everything out, then added things back 1-by-1.  I've only been working with MM a few days, and this works fine here.
